src/components/generate/index.js

```diff
@@ -0,0 +1,38 @@
+import { useState, useCallback } from 'react';
+import Input from '../input';
+import Select from '../select';
+import Button from '../button';
+import SnackBar from '../snackbar';
+import { selectOptions, isEmptyString } from '../utils/constants';
+
+const Generate = () => {
+    const [inputTextVal, setInputTextVal] = useState('');
+    const [validity, setValidity] = useState(selectOptions[0].name);
+    const [openSnackbar, setOpenSnackbar]=useState(false);
+    const [snackbarMessage, setSnackbarMessage] = useState('');
+    const [snackbarSeverity, setSnackbarSeverity] = useState('');
```

Comment:
create an enum (object.freeze) for severity and use it instead of manually typing the severity values

Reply:
Done